notebooks/2_uninformed_search/index.md

```diff
@@ -0,0 +1,255 @@
+
+# Uninformed Search
```

Comment:
Please explain what the term "Search" means exactly.

Reply:
You should explain in a way that everyone could read the article and understand the concept.